console-subscriber/examples/deep.rs

```diff
@@ -0,0 +1,44 @@
+use tokio::task;
+
+#[tokio::main]
```

Comment:
would be nice to have a doc comment somewhere in here explaining what this example demonstrates.